src/main/java/lotto/view/OutputView.java

```diff
@@ -0,0 +1,42 @@
+package lotto.view;
+
+import lotto.domain.Lotto;
+import lotto.domain.LottoRank;
+import lotto.domain.LottoResult;
+import lotto.domain.Lottos;
+
+import java.util.Arrays;
+
+public class OutputView {
+
+    public static void printLottos(Lottos lottos) {
+        System.out.println();
+        System.out.println(lottos.getCount() + "개를 구매했습니다.");
+        for (Lotto lotto : lottos.getLottos()) {
+            System.out.println(lotto.getNumbers());
+        }
+    }
+
+    public static void printResult(LottoResult lottoResult, int purchaseAmount) {
+        System.out.println();
+        System.out.println("당첨 통계");
```

Comment:
```suggestion
        System.out.println("\n당첨 통계");
```
개행문자를 추가하면 코드가 간결해질 거 같아요.